src/tabwidget.cpp

```diff
@@ -228,6 +229,10 @@ void TabWidget::renameTabsAfterRemove()
 
 void TabWidget::contextMenuEvent(QContextMenuEvent *event)
 {
+    int tabIndex = tabBar()->tabAt(tabBar()->mapFrom(this,event->pos()));
+    if (widget(tabIndex) == NULL) // did the user click on a tab
+        return;                   // if not return
```

Comment:
This is good -- the lack of it was a bug that caused a crash before -- but you could simply write:
```c++
    if (tabIndex == -1)
        return;
```
And please always use `nullptr` instead of `NULL` in a C++ code.

Reply:
I wasn't 100% sure of the default value